BSImagePicker.xcodeproj/project.pbxproj

```diff
@@ -41,7 +41,6 @@
 		55BCF8D521D52C1000386752 /* PhotoCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55BCF8C121D52C1000386752 /* PhotoCell.swift */; };
 		55BCF8D621D52C1000386752 /* AlbumCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55BCF8C221D52C1000386752 /* AlbumCell.swift */; };
 		55BCF8D721D52C1000386752 /* SelectionView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55BCF8C321D52C1000386752 /* SelectionView.swift */; };
```

Comment:
Decided to delete AlbumTitleView, as after the title sizing fix most of the code could be set during initialisation, and isn't needed when setting the title.